app/javascript/controllers/scroll_position_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from '@hotwired/stimulus'
+
+// Connects to data-controller="scroll-position"
+export default class extends Controller {
+  static targets = ['active']
+
+  connect () {
+    this.centerCurrentTalk()
+  }
+
+  centerCurrentTalk () {
+    if (!this.hasActiveTarget) return
+
+    const activeCard = this.activeTarget
+
+    const container = this.element
+    const containerHeight = container.clientHeight
+    const cardPosition = activeCard.offsetTop
+    const cardHeight = activeCard.clientHeight
+
+    // Calculate scroll position to center the card
+    const scrollPosition = cardPosition - (containerHeight / 2) + (cardHeight / 2)
+    container.scrollTop = Math.max(0, scrollPosition)
```

Comment:
Thoughts? 🙈 

```suggestion
    this.activeTarget.scrollIntoView({ block: "center" })
```

Reply:
Ah, I guess this doesn't work in all cases.

Reply:
not sure why when I first tried I had some issue, I tried again and it seems to work fine. I apply this change in took the opportunity to rename the controller scroll-into-view to make it more generic